src/nih_wayfinding/app/index.html

```diff
@@ -92,7 +92,24 @@ <h2 class="navbar-actionheader" ng-cloak ui-sref="profiles">{{ navbar.config.tit
     <script src="bower_components/angular-leaflet-directive/dist/angular-leaflet-directive.js"></script>
     <script src="bower_components/Leaflet.encoded/Polyline.encoded.js"></script>
     <script src="bower_components/turf/turf.min.js"></script>
+    <script src="bower_components/leaflet-plugins/control/Distance.js"></script>
```

Comment:
Yeah, grunt (bower?) keeps wanting to put them back in, which I didn't notice when I was setting it up. They should definitely be ignored.